docs/index.md

```diff
@@ -59,6 +61,101 @@ For example:
 - `LatencyAtQuantileMS(50.0) > 50`
 - `ResponseCodeRatio(500, 600, 0, 600) > 0.5`
 
+
+## <a id="launch"></a> Launch configuration
+
+Træfɪk can be configured using a TOML file configuration, arguments, or both.
+By default, Træfɪk will try to find a `traefik.toml` in the following places:
+- `/etc/traefik/`
+- `$HOME/.traefik/`
+- `.` the working directory
+
+You can override this by setting a `configFile` argument:
+
+```bash
+$ traefik --configFile=foo/bar/myconfigfile.toml
+```
+
+Træfɪk uses the following precedence order. Each item takes precedence over the item below it:
+
+- arguments
+- configuration file
+- default
+
+It means that arguments overrides configuration file.
```

Comment:
ok, so then, if I've set up something with a flag, and then update/add/change the value in the config file, it will not be taking into account until a restart (and a change in my flags) ; am I right ?